src/graphql/mutations/create_game_piece_actions.ts

```diff
@@ -102,7 +103,7 @@ async function handleMoveAction(
         moveTo: moveInput.moveTo
       }
     },
-    { transaction: transaction }
+    { transaction }
```

Comment:
Made this change all over as well.  These mean the same thing.  I used to prefer the explicitness of the original style, but it's pretty ubiquitous shorthand and definitely much cleaner once you get used to it.